apiserver/facades/client/modelupgrader/controllerupgrader.go

```diff
@@ -0,0 +1,154 @@
+package modelupgrader
+
+import (
+	"context"
+	"fmt"
+	"github.com/juju/juju/apiserver/common"
```

Comment:
You will probably have Static Analysis blowing up over the imports. They need to be grouped correctly and ordered.